src/app/clusters/commissioner-control-server/commissioner-control-server.h

```diff
@@ -0,0 +1,180 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace CommissionerControl {
+
+// Set to 64 bytes, which is more than enough for storing an IPv6 address in either binary or text form.
+static constexpr size_t kIpAddressBufferSize = 64;
```

Comment:
I don't get it.  Spec says this field is 4 or 16 bytes.  Where is the 64 coming from?

There is no "text form" involved; it's binary form only.